From 259ba1af92127599682265d0cc4bfef32f735349 Mon Sep 17 00:00:00 2001
From: Nicola Fontana <ntd@entidi.it>
Date: Fri, 13 Mar 2015 11:39:16 +0100
Subject: [PATCH 2/4] Export luaopen_lgi_corelgilua51

On windows functions must be exported from dynamic modules in order to
be accessible. Use the G_MODULE_EXPORT macro provided by GLib that
resolves to a no-op on non-windows platforms.
---
 lgi/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lgi/core.c b/lgi/core.c
index e5f44ca..429e877 100644
--- a/lgi/core.c
+++ b/lgi/core.c
@@ -654,7 +654,7 @@ set_resident (lua_State *L)
     }
 }
 
-int
+G_MODULE_EXPORT int
 luaopen_lgi_corelgilua51 (lua_State* L)
 {
   LgiStateMutex *mutex;
-- 
2.3.2

